commit 0eef9216e70d3a3bb78f93a79df0e99016af043d (refs/changes/89/2089/3)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-06-14T20:52:07+00:00 (2 years, 4 months ago)
    
    joyent/node-krill#11 contribution guidelines are missing
    joyent/node-krill#12 make check errors when run by Gerrit
    Reviewed by: Trent Mick <trent.mick@joyent.com>
    Reviewed by: Dave Pacheco <dap@joyent.com>
    Approved by: Dave Pacheco <dap@joyent.com>

diff --git a/CONTRIBUTING.md b/CONTRIBUTING.md
new file mode 100644
index 0000000..108eba2
--- /dev/null
+++ b/CONTRIBUTING.md
@@ -0,0 +1,19 @@
+# Contributing
+
+This repo __does not use GitHub pull requests__. Instead it uses
+[cr.joyent.us](https://cr.joyent.us) (Gerrit) for new changes. Anyone can submit
+changes. To get started, see the [cr.joyent.us user
+guide](https://github.com/joyent/joyent-gerrit/blob/master/docs/user/README.md).
+
+See the [Joyent Engineering
+Guidelines](https://github.com/joyent/eng/blob/master/docs/index.md) for general
+best practices expected in this repository.
+
+Contributions should be "make prepush" clean.  The "prepush" target runs the
+"check" target, which requires these separate tools:
+
+* https://github.com/davepacheco/jsstyle
+* https://github.com/davepacheco/javascriptlint
+
+If you're changing something non-trivial or user-facing, you may want to submit
+an issue first.
\ No newline at end of file
diff --git a/tests/tst.basic.js b/tests/tst.basic.js
index c6ee8c9..31bfcea 100644
--- a/tests/tst.basic.js
+++ b/tests/tst.basic.js
@@ -16,7 +16,7 @@ mod_assert.deepEqual([], pred.fields());
 mod_assert.deepEqual({}, pred.fieldsAndValues());
 mod_assert.equal('1', pred.toCStyleString());
 mod_assert.throws(function emptyPredToLDAPFilter() {
-    pred.toLDAPFilterString()
+    pred.toLDAPFilterString();
 }, /Cannot serialize empty predicate to LDAP search filter/);
 mod_assert.ok(pred.eval({}));
 mod_assert.ok(pred.eval({ 'hostname': 'sharptooth' }));
